recipe/build.sh

```diff
@@ -1,20 +1,8 @@
 #!/usr/bin/env bash
-set -e
+set -ex
 
 export CFLAGS="-O3"
 export CXXFLAGS="-O3"
```

Comment:
These should be appended instead of overridden

Reply:
Good point!